client/css/style.css

```diff
@@ -1492,18 +1504,16 @@ kbd {
 
 .context-menu-divider {
 	height: 1px;
-	margin: 6px 0;
 	background-color: rgba(0, 0, 0, .1);
 }
 
 .context-menu-item,
 .textcomplete-item {
 	cursor: pointer;
 	display: block;
-	padding: 4px 8px;
+	padding: 10px 8px;
 	color: #333;
-	margin-top: 6px;
-	margin-bottom: 6px;
```

Comment:
Um.. I didn't expect any groups here. This can be fixed only by returning to the old design. Are those groups a planned feature or I'm missing smth?